# Translation of OpenERP Server.
# This file contains the translation of the following modules:
# * pdi_connector
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-03-12 15:37+0000\n"
"PO-Revision-Date: 2014-03-12 16:40+0100\n"
"Last-Translator: Christophe CHAUVET <christophe.chauvet@gmail.com>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Translate Toolkit 1.3.0\n"

#. module: pdi_connector
#: help:pdi.instance.parameters,name:0
msgid "Name of the parameters, in upper case"
msgstr "Nom du paramètre, en Majuscule"

#. module: pdi_connector
#: view:pdi.task:0
msgid "Information"
msgstr "Information"

#. module: pdi_connector
#: view:pdi.instance:0
msgid "PDI Instance"
msgstr "Instance PDI"

#. module: pdi_connector
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_configuration_menu
msgid "Pentaho Data Integration"
msgstr "Pentaho Data Integration"

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v5.0"
msgstr "v5.0"

#. module: pdi_connector
#: field:pdi.instance,trans_ids:0
msgid "Transformations"
msgstr "Transformations"

#. module: pdi_connector
#: view:pdi.instance:0
#: field:pdi.instance,param_ids:0
#: view:pdi.instance.parameters:0
#: view:pdi.task:0
#: field:pdi.task,param_ids:0
#: view:pdi.transformation:0
#: field:pdi.transformation,param_ids:0
msgid "Parameters"
msgstr "Paramètres"

#. module: pdi_connector
#: help:pdi.task,note:0
#: help:pdi.transformation,note:0
msgid "Explain the process for the user"
msgstr "Expliquer le processus à l'utilisateur"

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_base_config_settings
msgid "base.config.settings"
msgstr "base.config.settings"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:362
#: code:addons/pdi_connector/pdi.py:628
#, python-format
msgid "(%s) Unknown error"
msgstr "(%s) Erreur inconnue"

#. module: pdi_connector
#: field:base.config.settings,module_pdi_connector:0
msgid "Active PDI module"
msgstr "Activer le module PDI"

#. module: pdi_connector
#: model:res.groups,name:pdi_connector.group_pdi_user
msgid "PDI / User"
msgstr "PDI / Utilisateur"

#. module: pdi_connector
#: field:pdi.instance,note:0
#: field:pdi.task,note:0
#: field:pdi.transformation,note:0
msgid "Note"
msgstr "Note"

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Debug"
msgstr "Debogage"

#. module: pdi_connector
#: view:pdi.instance:0
msgid "Real login in OpenERP"
msgstr "Identifiant réel d'OpenERP"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:622
#, python-format
msgid "(7) The task couldn't be loaded from XML or the Repository"
msgstr "(7) La tache ne peut être charger depuis le XML ou le dépôt"

#. module: pdi_connector
#: view:pdi.task:0
#: view:pdi.transformation:0
msgid "Debug information"
msgstr "Information de debuggage"

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Basic"
msgstr "Basique"

#. module: pdi_connector
#: help:pdi.instance,username:0
msgid "OpenERP User to connect tools with XMLRPC or NETRPC"
msgstr "Utilisateur OpenERP pour connecter des outils en XMLRPC ou NETRPC"

#. module: pdi_connector
#: model:ir.actions.act_window,name:pdi_connector.act_open_pdi_instance_view
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_instance
#: view:pdi.instance:0
#: field:pdi.instance.parameters,instance_id:0
#: view:pdi.task:0
#: field:pdi.task,instance_id:0
#: view:pdi.transformation:0
#: field:pdi.transformation,instance_id:0
msgid "Instance"
msgstr "Instance"

#. module: pdi_connector
#: model:ir.actions.act_window,name:pdi_connector.act_open_pdi_transformation_view
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_transformation
#: view:pdi.instance:0
#: field:pdi.trans.param,trans_id:0
#: view:pdi.transformation:0
msgid "Transformation"
msgstr "Transformation"

#. module: pdi_connector
#: model:ir.module.category,name:pdi_connector.module_category_etl_connector
msgid "ETL"
msgstr "ETL"

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v3.2"
msgstr "v3.2"

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Rowlevel"
msgstr "Niveau par ligne"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:350
#: code:addons/pdi_connector/pdi.py:618
#, python-format
msgid "(1) Errors occurred during processing"
msgstr "(1) Une erreur est apparu lors du processus"

#. module: pdi_connector
#: help:pdi.transformation,cron_id:0
msgid "If fill, this transformation is schedule"
msgstr "Si rempli la transformation est planifiée"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:620
#, python-format
msgid "(2) An unexpected error occurred during loading / running of the task"
msgstr "(2) Une erreur est apparu lors du chargement / lancement de la tâche"

#. module: pdi_connector
#: field:pdi.task,log_cmd:0
#: field:pdi.transformation,log_cmd:0
msgid "Log Command"
msgstr "Tracer la commande"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:354
#, python-format
msgid "(3) Unable to prepare and initialize this transformation"
msgstr "(3) Imossible de préparer ou d'initialiser cette transformation"

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_instance_parameters
msgid "Parameters for this instance"
msgstr "Paramètres pour cette instance"

#. module: pdi_connector
#: field:pdi.task,level:0
#: field:pdi.transformation,level:0
msgid "Level"
msgstr "Niveau de traces"

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v4.2"
msgstr "v4.2"

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v4.0"
msgstr "v4.0"

#. module: pdi_connector
#: help:pdi.instance,repo_pass:0
msgid "Enter the password for this repository"
msgstr "Entrer le mot de passe pour ce dépôt"

#. module: pdi_connector
#: selection:pdi.task,state:0
#: selection:pdi.transformation,state:0
msgid "Running"
msgstr "En cours d'exécution"

#. module: pdi_connector
#: help:pdi.task,log_cmd:0
#: help:pdi.transformation,log_cmd:0
msgid "Log command file as info, usefull for debugging"
msgstr "Tracer la commande dans le journal en mode normal, utile pour le debuggage"

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v4.4"
msgstr "v4.4"

#. module: pdi_connector
#: model:res.groups,name:pdi_connector.group_pdi_manager
msgid "PDI / Manager"
msgstr "PDI / Responsable"

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Nothing"
msgstr "Rien du tout"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:356
#, python-format
msgid "(7) The transformation couldn't be loaded from XML or the Repository"
msgstr "(7) La transformation ne peut être charger du XML ou du dépôt"

#. module: pdi_connector
#: field:pdi.instance,password:0
#: field:pdi.instance,repo_pass:0
msgid "Password"
msgstr "Mot de passe"

#. module: pdi_connector
#: help:pdi.instance.parameters,value:0
#: help:pdi.task.param,value:0
#: help:pdi.trans.param,value:0
msgid "Use [[ ]] to eval, time, datetime, timedelta is available"
msgstr "Utiliser [[ ]] pour évaluer, time, datetime, timedelta sont disponibles"

#. module: pdi_connector
#: view:pdi.instance:0
#: view:pdi.task:0
#: view:pdi.transformation:0
msgid "Notes"
msgstr "Notes"

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v4.3"
msgstr "v4.3"

#. module: pdi_connector
#: field:pdi.instance,repo_user:0
#: field:pdi.instance,username:0
msgid "Username"
msgstr "Nom d'utilisateur"

#. module: pdi_connector
#: field:pdi.task,state:0
#: field:pdi.transformation,state:0
msgid "Status"
msgstr "État"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:406
#, python-format
msgid ""
"\n"
"BE CAREFULL, record are incorrect for %s"
msgstr ""
"\n"
"ATTENTION, les enregistrements sont incorrectes pour %s"

#. module: pdi_connector
#: field:pdi.instance,name:0
#: field:pdi.instance,repo_name:0
#: field:pdi.instance.parameters,name:0
#: field:pdi.task,name:0
#: field:pdi.task.param,name:0
#: field:pdi.trans.param,name:0
#: field:pdi.transformation,name:0
msgid "Name"
msgstr "Nom"

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_task_param
msgid "Parameter to launch task"
msgstr "Paramètre de la tâche à lancer"

#. module: pdi_connector
#: view:pdi.instance:0
#: view:pdi.task:0
#: view:pdi.transformation:0
msgid "Launch"
msgstr "Lancer"

#. module: pdi_connector
#: help:pdi.instance,password:0
msgid "Password for the OpenERP User"
msgstr "Mot de passe de l'utilisateur OpenERP"

#. module: pdi_connector
#: help:pdi.instance,repo_user:0
msgid "Enter the username for this repository"
msgstr "Entrer le nom d'utilisateur pour ce dépôt"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:258
#: code:addons/pdi_connector/pdi.py:531
#, python-format
msgid "pdi path does not exist"
msgstr "Le chemin vers PDI n'existe pas"

#. module: pdi_connector
#: field:pdi.instance.parameters,value:0
#: field:pdi.task.param,value:0
#: field:pdi.trans.param,value:0
msgid "Value"
msgstr "Valeur"

#. module: pdi_connector
#: model:res.groups,comment:pdi_connector.group_pdi_menu
msgid ""
"Use to display the kettle as root menu\n"
"select another group (user or manager) to user the transformation or task"
msgstr ""
"Utiliser pour afficher le menu racine de Kettle\n"
"Sélection un autre groupe (utilisateur ou responsable) pour utiliser la transformation ou tâche"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:360
#: code:addons/pdi_connector/pdi.py:626
#, python-format
msgid "(9) Command line usage printing"
msgstr "(9) Impression des informations de ligne de commande"

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_transformation
msgid "Transformation of a PDI instance"
msgstr "Tâche d'un instance PDI"

#. module: pdi_connector
#: view:pdi.transformation:0
msgid "Install Cron"
msgstr "Installer tâche planifiée"

#. module: pdi_connector
#: selection:pdi.task,state:0
#: selection:pdi.transformation,state:0
msgid "Stopped"
msgstr "Arreté"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:258
#: code:addons/pdi_connector/pdi.py:531
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
#, python-format
msgid "Error"
msgstr "Erreur"

#. module: pdi_connector
#: field:pdi.task,directory:0
#: field:pdi.transformation,directory:0
msgid "Directory"
msgstr "Répertoire"

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_task
msgid "Task of a PDI instance"
msgstr "Instance PDI de la tâche"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:352
#, python-format
msgid "(2) An unexpected error occurred during loading / running of the transformation"
msgstr "(2) Une erreur est apparue lors du chargement / lancement de la transformation"

#. module: pdi_connector
#: selection:pdi.task,level:0
#: selection:pdi.transformation,level:0
msgid "Detailed"
msgstr "Détaillé"

#. module: pdi_connector
#: view:pdi.transformation:0
msgid "Informations"
msgstr "Informations"

#. module: pdi_connector
#: view:pdi.instance:0
#: field:pdi.instance,task_ids:0
msgid "Tasks"
msgstr "Tâches"

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_instance
msgid "Instance of PDI server"
msgstr "Instance du serveur PDI"

#. module: pdi_connector
#: help:pdi.task,memory:0
#: help:pdi.transformation,memory:0
msgid "Custom memory to launch this treament, if 0 use standard"
msgstr "Personnalisé la mémoire pour lancer ce traitement, si 0 on utilise le réglage standard"

#. module: pdi_connector
#: view:pdi.task:0
#: view:pdi.transformation:0
msgid "Group By..."
msgstr "Grouper par..."

#. module: pdi_connector
#: selection:pdi.instance,version:0
msgid "v4.1"
msgstr "v4.1"

#. module: pdi_connector
#: model:ir.model,name:pdi_connector.model_pdi_trans_param
msgid "Parameter to launch transformation"
msgstr "Paramètres de la transformation à lancer"

#. module: pdi_connector
#: view:pdi.transformation:0
msgid "Uninstall Cron"
msgstr "Désinstaller tâche planifiée"

#. module: pdi_connector
#: code:addons/pdi_connector/pdi.py:358
#: code:addons/pdi_connector/pdi.py:624
#, python-format
msgid "(8) Error loading steps or plugins (error in loading one of the plugins mostly)"
msgstr "(8) Erreur lors du chargement de l'étapes ou du plugin (erreur au chargement d'un des plugins le plus souvent)"

#. module: pdi_connector
#: view:pdi.instance:0
#: view:pdi.task:0
#: view:pdi.transformation:0
msgid "General"
msgstr "Général"

#. module: pdi_connector
#: field:pdi.transformation,cron_id:0
msgid "Cron"
msgstr "Tâche planifiée"

#. module: pdi_connector
#: help:base.config.settings,module_pdi_connector:0
msgid "Install Pentaho Data Integration Connector for OpenERP"
msgstr "Installe le connecteur Pentaho Data Integration pour OpenERP "

#. module: pdi_connector
#: field:pdi.instance,version:0
msgid "Version"
msgstr "Version"

#. module: pdi_connector
#: field:pdi.task,memory:0
#: field:pdi.transformation,memory:0
msgid "Memory"
msgstr "Mémoire"

#. module: pdi_connector
#: view:pdi.instance:0
msgid "OpenERP for external integration"
msgstr "OpenERP pour une intégration externe"

#. module: pdi_connector
#: view:pdi.instance:0
msgid "Repository information"
msgstr "Information du dépôt"

#. module: pdi_connector
#: view:pdi.transformation:0
msgid "Scheduling"
msgstr "Planification"

#. module: pdi_connector
#: model:ir.actions.act_window,name:pdi_connector.act_open_pdi_task_view
#: model:ir.ui.menu,name:pdi_connector.menu_pdi_task
#: view:pdi.instance:0
#: view:pdi.task:0
#: field:pdi.task.param,trans_id:0
msgid "Task"
msgstr "Tâche"

#. module: pdi_connector
#: view:base.config.settings:0
msgid "Penthao Data Inetgration"
msgstr "Pentaho Data Integration"

#. module: pdi_connector
#: model:res.groups,name:pdi_connector.group_pdi_menu
msgid "PDI / Menu"
msgstr "PDI / Menu"

#. module: pdi_connector
#: help:pdi.instance,repo_name:0
msgid "Enter the name of the repository"
msgstr "Entrer le nom du dépôt"

#~ msgid "pourcent(username)s : name of the session user"
#~ msgstr "pourcent(username)s : nom de l'utilisateur de la session"
#~ msgid "pourcent(dbname)s : for database name"
#~ msgstr "pourcent(dbname)s : pour le nom de base de données"
#~ msgid "Value of the parameter"
#~ msgstr "Valeur du paramètre"
#~ msgid "PDI Configuration"
#~ msgstr "Configuration de PDI"
#~ msgid "Legend"
#~ msgstr "Légende"
#~ msgid "pourcent(uid)s : for id of username"
#~ msgstr "pourcent(uid)s : Pour l'ID de l'utilisateur"
#~ msgid "PDI - ETL"
#~ msgstr "PDI - ETL"
#~ msgid "pourcent(dbuser)s : for database username"
#~ msgstr "pourcent(dbuser)s : pour le nom de la base de données"
#~ msgid "PDI Transformation"
#~ msgstr "Transformation PDI"
#~ msgid "PDI Task"
#~ msgstr "Tâche PDI"
#~ msgid ""
#~ "Module to manage Pentaho Data Integration\n"
#~ "    - Start task or transformation define on the repository\n"
#~ "    - Schedule a task\n"
#~ "    "
#~ msgstr ""
#~ "Module de gestion de Pentaho Data Integration\n"
#~ "    - Lance une tâche ou une transformation depuis le dépôt\n"
#~ "    - Planifie une tâche\n"
#~ msgid "%(username)s : name of the session user"
#~ msgstr "%%(username)s : Nom de la session utilisateur"
#~ msgid "%(dbuser)s : for database username"
#~ msgstr "%%(dbuser)s : Pour l'utilisateur de la base de données"
#~ msgid "%(uid)s : for id of username"
#~ msgstr "%%(uid)s : pour l'identifiant de l'utilisateur"
#~ msgid "PDI Connector"
#~ msgstr "Connecteur Pentaho-Data-Integration"
#~ msgid "%(dbname)s : for database name"
#~ msgstr "%%(dbname)s : pour le nom de la base de données"
#~ msgid "The Object name must start with x_ and not contain any special character !"
#~ msgstr "Le nom de l'objet ne doit pas commencer oar x_ et ne pas contenir de caractère spéciaux !"
#~ msgid "Invalid XML for View Architecture!"
#~ msgstr "XML de la vue non valide!"
#~ msgid "Invalid model name in the action definition."
#~ msgstr "Nom de modèle invalide pour cette action définie"

